backend/app/models/user.rb

```diff
@@ -6,4 +6,6 @@ class User < ActiveRecord::Base
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
   include DeviseTokenAuth::Concerns::User
+
+  has_many :journals
```

Comment:
[imo]
cc: @yuki-snow1823 
ユーザが退会して削除された際、紐づく記事は削除されない仕様ということで良さですかね🙋‍♂️
```suggestion
  has_many :journals, dependent: :destroy
```

Reply:
@besumero628 @shikata1112 
削除されない方針でいいと思います！退会がほとんど可能性ないですが、ナレッジとか日記とかは振り返れたほうが良さそうと思いました。

Reply:
@yuki-snow1823 @shikata1112 
私もその対応で良いかと思います！